src/program/snabbvmx/README

```diff
@@ -0,0 +1,7 @@
+Usage:
```

Comment:
A few words about what snabbvmx is and does would be helpful, see for instance the snabbnfv `README.md` file.